SYCL/KernelAndProgram/cache_env_vars_lin.cpp

```diff
@@ -1,6 +1,7 @@
 // No JITing for host devices and diffrent environment variables on linux and
 // windows.
 // REQUIRES: (level_zero || opencl) && linux
+// XFAIL: *
```

Comment:
here and in other places - would be good to add a comment why XFAIL and what needs to happen to re-enable it